common/lib/xmodule/xmodule/x_module.py

```diff
@@ -18,9 +18,9 @@
 from webob.multidict import MultiDict
 from lazy import lazy
 
-from xblock.core import XBlock, XBlockAside
+from xblock.core import XBlock, XBlockAside, XBlockMixin
 from xblock.fields import (
-    Scope, Integer, Float, List, XBlockMixin,
```

Comment:
`xblock` monkey-patches it in from `__init__`, for backwards compatibility. It's super gross.